### PR TITLE
[IMPORTANT] Add missing year and Apple INC. in license

### PR DESCRIPTION
**Pull request by KittyBorgX**
The LICENSE in this repository had a missing Year and org. This PR adds the relavant date and Apple INC. org to the license

### CODE REVIEW

**Comment by KittyBorgX** [comment]
@swift-ci Please smoke test

**Comment by KittyBorgX** [comment]
This is my first time contributing! I'm requesting a rewiew from one of the Owners! [ As mentioned in https://github.com/apple/swift/blob/main/docs/HowToGuides/FirstPullRequest.md ]

@airspeedswift

File: LICENSE.txt [hunk 1]
```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
```
Comment (Kyle-Ye) [inline LICENSE.txt:187]:
```suggestion
     // You should remove the below lines too.
```

Comment (KittyBorgX) [reply LICENSE.txt]: Thanks for that! I've removed the appendix bit now!

**Comment by Kyle-Ye** [comment]
LGTM. 

One more thing is that I don't know how Apple thinks the name: Should it be "Apple INC.", "Apple, Inc" or "APPLE INC.".

**Comment by KittyBorgX** [comment]
@Kyle-Ye Judging the contents of this file : 
https://github.com/apple/password-manager-resources/blob/main/LICENSE.md

Im guessing it would be Apple Inc. or Apple INC.

**Comment by xwu** [comment]
Hi @KittyBorgX, thanks for looking to contribute.

What you've edited is the Apache license text. The appendix here is information for readers about how to apply the license to their own work, and it's not supposed to have Apple's name here. Indeed, Apple doesn't have any ownership over the Apache license itself, and you're not supposed to say that your own work belongs to Apple when you apply the Apache license.

**Pull request closed without merging**